### PR TITLE
feat(workflow): add similar color spectrum

### PR DESCRIPTION
**Pull request by billyvg**
Adds a similar score color spectrum in the similar issues view. Refactors existing components to re-use a LESS mixin for colors.

image

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/components/similarSpectrum.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/components/similarSpectrum.jsx]: I think we need to start introducing some subfolders inside `/components` to help organize these things (I feel like there's 100+ files in `/components/*`).

File: src/sentry/static/sentry/app/components/similarSpectrum.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  },
+
+  render() {
+    let {className} = this.props;
+    let cx = classNames('similar-spectrum', className);
+
+    return (
+      <div className={cx}>
+        <span>
+          Similar
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/components/similarSpectrum.jsx]: Translatable?

File: src/sentry/static/sentry/less/palette.less [hunk 3]
```diff
@@ -96,3 +96,22 @@
...
+  }
+  &.med {
+    background-color: @yellow-orange;
+  }
+  &.med-high {
+    background-color: mix(@yellow-orange, @green, 40%);
+  }
+  &.high {
+    background-color: @green;
+  }
```
Comment (benvinegar) [inline src/sentry/static/sentry/less/palette.less]: Do we mean to have similarity-keywords inside `palette.less`? I'd expect this file to just be colors/variables, and similarity-specific declarations would be in `similarity.less` or something.

Just some thoughts, doesn't require any changes if you disagree.

**Review by benvinegar — APPROVED** [review]

File: src/sentry/static/sentry/app/components/similarSpectrum.jsx (hunk 1 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/components/similarSpectrum.jsx]: Yeah, alternatively we may want to think structure on a broader level (i.e. folders per feature). Burying components within subfolders doesn't necessarily help. I'm guessing many of the components inside of `/components/` are only used once.

**Pull request merged**